shell/common/fixtures/shell_test.dart

```diff
@@ -25,6 +25,7 @@ void reportTimingsMain() {
       }
     }
     nativeReportTimingsCallback(timestamps);
+    PlatformDispatcher.instance.onReportTimings = (List<FrameTiming> timings) {};
```

Comment:
Specifically here - we cannot get called again on the raster thread in here, since we synchronously reset the callback to a no-op after the first call.